tests/integration.rs

```diff
@@ -0,0 +1,70 @@
+// Copyright 2024, Horizen Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use risc0_verifier::{verify, VerifyError};
+use rstest::rstest;
+use serde::Deserialize;
+use std::path::{Path, PathBuf};
+
+fn load_data(path: &Path) -> (Vec<u8>, [u32; 8]) {
+    #[derive(Deserialize)]
+    struct Data {
+        raw_proof_data: String,
+        image_id_data: [u32; 8],
+    }
+
+    let Data {
+        raw_proof_data,
+        image_id_data,
+    } = serde_json::from_reader(std::fs::File::open(path).unwrap()).unwrap();
+
+    let proof_raw_data = <Vec<u8>>::try_from(hex::decode(raw_proof_data).unwrap()).unwrap();
+
+    (proof_raw_data, image_id_data)
+}
+
+#[rstest]
+fn should_verify_valid_proof(#[files("./resources/*.json")] path: PathBuf) {
```

Comment:
Minuscule thing here; what about the following, it might help with readability?
```suggestion
fn should_verify_valid_proof(#[files("./resources/valid_proof_*.json")] path: PathBuf) {
```

Reply:
Modified as suggested.